ast/check.go

```diff
@@ -95,12 +95,10 @@ func (tc *typeChecker) Env(builtins map[string]*Builtin) *TypeEnv {
 // are found. The resulting TypeEnv wraps the provided one. The resulting
 // TypeEnv will be able to resolve types of vars contained in the body.
 func (tc *typeChecker) CheckBody(env *TypeEnv, body Body) (*TypeEnv, Errors) {
-
```

Comment:
While I don't mind this change personally, I think these newlines after the function signature are intentional and the prefered style for the OPA project. Either way, it would be good if preferences like this could be codified. I don't think `go fmt` allows that, but maybe there are other tools we could use.